lib/welcome.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'customer'
+require_relative 'takeaway'
+require_relative 'message_handler'
+require_relative 'menu'
+require_relative 'order'
+require_relative 'dishes'
+
+#>> IRB UI will be welcome.customer.check_menu (under hood calls takeaway)
+class Welcome
```

Comment:
Again, this class is only necessary if we're tying the presentation of the system to terminal. (this will make more sense after this week).